sdks/python/container/piputil.go

```diff
@@ -51,7 +51,7 @@ func pipInstallRequirements(files []string, dir, name string) error {
 			// used without following their dependencies.
 			args := []string{"install", "-r", filepath.Join(dir, name), "--disable-pip-version-check", "--no-index", "--no-deps", "--find-links", dir}
 			if err := execx.Execute(pip, args...); err != nil {
-				return err
+		        fmt.Println("Requirements cache " + dir + " is empty. Downloading packages from PyPI")
```

Comment:
Printing this in case if the first Pip call fails(when user skips populating requirements cache) to provide more context if user looks into the logs